File: Vesti/Vesti/ArticleTableViewCell.m
//
//  ArticleTableViewCell.m
//  Vesti
//
//  Created by Djuro Alfirevic on 4/26/16.
//  Copyright © 2016 Djuro Alfirevic. All rights reserved.
//

#import "ArticleTableViewCell.h"

@implementation ArticleTableViewCell

#pragma mark - Properties

- (void)setArticle:(Article *)article {
    _article = article;
    
    self.articleTitleLabel.text = article.title;
    self.asyncImageView.imageURL = article.imageURL;
}

@end

File: VestiS/VestiS/ArticleTableViewCell.swift
//
//  ArticleTableViewCell.swift
//  VestiS
//
//  Created by Djuro Alfirevic on 8/3/16.
//  Copyright © 2016 Djuro Alfirevic. All rights reserved.
//

import UIKit

class ArticleTableViewCell: UITableViewCell {

    // MARK: - Properties
    @IBOutlet weak var asyncImageView: AsyncImageView!
    @IBOutlet weak var titleLabel: UILabel!
    var article: Article? {
        didSet {
            if let title = article?.title {
                titleLabel.text = title
            }
            
            if let url = article?.imageURL {
                asyncImageView.imageURL = url
            }
        }
    }
    
}